### PR TITLE
Azure US Government - LA & AI

### PR DESCRIPTION
**Pull request by Akreddy123**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.Framework/Helpers/WebRequestHelper.ps1 [hunk 1]
```diff
@@ -15,9 +15,53 @@ class WebRequestHelper {
...
+		else {
+			return "https://dc.services.visualstudio.com/v2/track"
+		}
+	}
+
+	hidden static [string] GetLADataCollectorAPI()
+	{
+		$rmContext = [ContextHelper]::GetCurrentRMContext();
+		$azureEnv= $rmContext.Environment.Name 
+		if($azureEnv = "AzureUSGovernment")
```
Comment (VishalHaibatpure) [inline src/AzSK.Framework/Helpers/WebRequestHelper.ps1]: use -eq

File: src/AzSK.Framework/Helpers/WebRequestHelper.ps1 [hunk 2]
```diff
@@ -15,9 +15,53 @@ class WebRequestHelper {
...
 	
+	hidden static [string] GetApplicationInsightsEndPoint()
+	{
+		$rmContext = [ContextHelper]::GetCurrentRMContext();
+		$azureEnv= $rmContext.Environment.Name 
+		if($azureEnv = "AzureUSGovernment")
+		{
+            return "https://dc.applicationinsights.us/v2/track"
+		}
+		elseif ($azureEnv = "AzureChinaCloud" ) {
```
Comment (VishalHaibatpure) [inline src/AzSK.Framework/Helpers/WebRequestHelper.ps1]: use -eq

File: src/AzSK.Framework/Helpers/WebRequestHelper.ps1 [hunk 3]
```diff
@@ -15,9 +15,53 @@ class WebRequestHelper {
         return [WebRequestHelper]::InvokeGetWebRequest($uri, [WebRequestHelper]::GetAuthHeaderFromUri($uri));
 	}
 	
+	hidden static [string] GetApplicationInsightsEndPoint()
```
Comment (VishalHaibatpure) [inline src/AzSK.Framework/Helpers/WebRequestHelper.ps1]: One generic method for getting endpoints(AppInsight,DatacollectorAPI,GraphUrl) url with mapping in some json file with environments would have done better design here. We are almost repeating same code block in every method

Comment (Akreddy123) [reply src/AzSK.Framework/Helpers/WebRequestHelper.ps1]: We just have 2 endpoints here. Next time, If there is any other endpoint to be added, will follow that approach.

File: src/AzSK.Framework/Helpers/WebRequestHelper.ps1 (hunk 1 above)
Comment (Akreddy123) [reply src/AzSK.Framework/Helpers/WebRequestHelper.ps1]: Done

File: src/AzSK.Framework/Helpers/WebRequestHelper.ps1 (hunk 2 above)
Comment (Akreddy123) [reply src/AzSK.Framework/Helpers/WebRequestHelper.ps1]: Done

**Pull request merged**